mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.deleteOrder = async (req, res) => {
+  try {
+    const order = await Order.findByIdAndRemove(req.params.orderId);
+    if (!order) {
+      return res.status(404).json({ message: "Order does not exist" });
+    }
+    res.status(200).json({ message: "Order deleted successfully" });
```

Comment:
The response status code and message should be consistent throughout the application. In this case, the success response message is not following the standard format used in other parts of the code. It should be changed to `res.json({ success: true, message: "Order deleted successfully" });` to maintain consistency.